React/wbooks/src/Login/style.css

```diff
@@ -0,0 +1,131 @@
+* {
```

Comment:
Don't commit the css. Remove this and any other css file from the index and add *.css in the `.gitignore`.